client/package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.5.0",
     "axios": "^0.21.1",
+    "bootstrap": "^4.5.3",
```

Comment:
Quick note when doing imports and installs into your system. Note the `^` before the version numbers, this denotes to the package installer to look for the most recent version of the package which can lead to issues in the future when your app installs some app breaking change. So I would remove all the `^` in your package.json and do a new `npm install`